### ISSUE TITLE
RunTime Error: PDF could not be generated! Command Error: /Users/.../wkhtmltopdf:21:in `system': Interrupt

### ISSUE THREAD

**Issue by rycwilson** [mileszs/wicked_pdf#552]
I'm trying to get a very simple test case fired up.  For starters not including any static assets.  I gather this isn't a wicked_pdf issue, but a PATH issue (or something) related to the wkhtmltopdf binary.  Logs below.

My experience is limited, so forgive me if I've missed an obvious step.  Thanks for any insight!

In my Gemfile (no change in the error if I point to github):
```
gem 'wicked_pdf'
gem 'wkhtmltopdf-binary'
```

``` which wkhtmltopdf ``` =>  ```/Users/wilson/.rbenv/shims/wkhtmltopdf ``` 

(Not clear to me why this doesn't exactly jibe with the longer path appearing after the Command Error below)

I've tried uncommenting these in wicked_pdf.rb (one at a time of course):
```exe_path: '/Users/wilson/.rbenv/shims/wkhtmltopdf'```
```exe_path: Gem.bin_path('wkhtmltopdf-binary', 'wkhtmltopdf')```

```
RuntimeError - Error: Failed to execute:
["/Users/wilson/.rbenv/shims/wkhtmltopdf", "-q", "file:////var/folders/zl/c4p4mn894zv0jkrjy6sb817h0000gn/T/wicked_pdf20160627-5969-16fnopl.html", "/var/folders/zl/c4p4mn894zv0jkrjy6sb817h0000gn/T/wicked_pdf_generated_file20160627-5969-ccsisc.pdf"]
Error: PDF could not be generated!
 Command Error: /Users/wilson/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/wkhtmltopdf-binary-0.9.9.3/bin/wkhtmltopdf:21:in `system': Interrupt
	from /Users/wilson/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/wkhtmltopdf-binary-0.9.9.3/bin/wkhtmltopdf:21:in `<top (required)>'
	from /Users/wilson/.rbenv/versions/2.2.2/bin/wkhtmltopdf:23:in `load'
	from /Users/wilson/.rbenv/versions/2.2.2/bin/wkhtmltopdf:23:in `<main>':
  wicked_pdf (1.0.6) lib/wicked_pdf.rb:62:in `rescue in pdf_from_string'
  wicked_pdf (1.0.6) lib/wicked_pdf.rb:65:in `pdf_from_string'
```

**Comment by unixmonkey** [comment]
Could you try commenting out both `exe_path` lines and letting wicked_pdf figure out where the binary is?

**Comment by rycwilson** [comment]
Yes, I've tried that as well.  Same error message.

Only thing in my wicked_pdf.rb is ```layout: 'pdf.html.erb'```

Also, not sure if relevant, but I've added these lines to application.rb:
```require 'wicked_pdf'```
```config.middleware.use WickedPdf::Middleware```

... so that I can simply append the url with ```.pdf```

Looking at those instructions again, I may be doing it wrong, because I've not excluded the action I'm testing, which does contain the standard ```render pdf: 'some_pdf'``` statement.  But again, not sure of relevance to the error message at hand.
